README.md

```diff
@@ -51,7 +51,8 @@ the fun stuff come.
 
 ## Android setup
 
-You'll need the usual ANDROID_NDK_HOME, and `adb` somewhere in your path.
+You'll need the usual ANDROID_NDK_HOME, and `adb` somewhere in your path. fb-ad
```

Comment:
you missed the b in fb-adb and you should enclose the executable name with backquotes